gobblin-compaction/src/main/java/org/apache/gobblin/compaction/verify/InputRecordCountHelper.java

```diff
@@ -123,7 +125,8 @@ public void saveState (Path dir, State state) throws IOException {
     saveState(this.fs, dir, state);
   }
 
-  private static void saveState (FileSystem fs, Path dir, State state) throws IOException {
+  @VisibleForTesting
```

Comment:
let's do package static instead of public static if visible for testing purpose?

Reply:
It's because test class and InputRecordCountHelper are not in the same package